internal/testutil/acc/name.go

```diff
@@ -36,3 +40,17 @@ func RandomIP(a, b, c byte) string {
 func RandomEmail() string {
 	return fmt.Sprintf("%s-%s@mongodb.com", prefixName, acctest.RandString(10))
 }
+
+func ProjectIDGlobal(tb testing.TB) string {
+	tb.Helper()
+	return projectID(tb, prefixProjectKeep+"-global")
+}
+
+func projectID(tb testing.TB, name string) string {
+	tb.Helper()
+	SkipInUnitTest(tb)
+	resp, _, _ := ConnV2().ProjectsApi.GetProjectByName(context.Background(), name).Execute()
+	id := resp.GetId()
+	require.NotEmpty(tb, id)
+	return id
```

Comment:
```suggestion
	require.NotEmpty(tb, id, fmt.Sprintf("Project name not found: %s", name))
```

Reply:
i normally prefer not to add descriptions to asserts / requires, etc., as it's easy that they get outdated or that they're redundant, and in case that this fails you have the stack trace and it's very easy to see what happened

Reply:
in this case it's true that it can be useful to know the project name that failed.   i've added your suggestion, thx